pkg/controllers/kube-apiserver.go

```diff
@@ -90,7 +92,9 @@ func (s *KubeAPIServer) configure(cfg *config.MicroshiftConfig) error {
 	s.kubeconfig = filepath.Join(cfg.DataDir, "resources", "kube-apiserver", "kubeconfig")
 	s.verbosity = cfg.LogVLevel
 
-	caCertFile := filepath.Join(cfg.DataDir, "certs", "ca-bundle", "ca-bundle.crt")
+	certsDir := cryptomaterial.CertsDirectory(cfg.DataDir)
+	caCertFile := filepath.Join(certsDir, "ca-bundle", "ca-bundle.crt")
```

Comment:
Why isn't there a helper function for the client CA bundle certficate path?

Reply:
I didn't figure out what the `ca-bundle` directory stands for yet so I did not know how to name it.

Reply:
I just mean that `cryptomaterial.TotalClientCABundlePath` builds a path to the `client-ca.crt`. Why isn't there a helper function to build the path to `ca-bundle.crt`?

Reply:
I did not want to go and replace the reference everywhere, since we will be trying to get rid of this bundle :slightly_smiling_face:

But I can add a helper for that, at least we'll be able to find all the spots where it is used more easily later.